### PR TITLE
Added skeleton for GUI.

### PR DESCRIPTION
**Pull request by Eskalior**
- added required c# classes with minimal content
- added xml file containing dummy elements for the trait selection window
- added rule to build 
- added main menu option for testing the window

### CODE REVIEW

File: TraitsExpanded/GUI/TraitSelectionGauntletScreen.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using TaleWorlds.Engine.GauntletUI;
+using TaleWorlds.Engine.Screens;
+using TaleWorlds.Library;
+
+namespace TraitsExpanded.GUI
+{
+    class TraitSelectionGauntletScreen : ScreenBase
+    {
+		private GauntletLayer _gauntletLayer;
```
Comment (ghost) [inline TraitsExpanded/GUI/TraitSelectionGauntletScreen.cs:9]: please convert tabs to spaces

**Review by ghost — CHANGES_REQUESTED** [review]
looks good other than the inconsistent tabs vs spaces

**Comment by MatasV** [comment]
Looks good other than the extra spaces in a couple of lines

**Review by MatasV — APPROVED** [review]
I don't think a couple of tabs is a showstopper. He'll change it on the next iteration

**Pull request merged**